src/views/players/PlayerView/PlayerView.vue

```diff
@@ -198,7 +198,7 @@
 
     synthesized: SynthesizedDesign[] = [];
 
-    myAchievements: {[name: string]: ProfileAchievement} = {};
+    achievements: {[name: string]: ProfileAchievement} = {};
```

Comment:
Why is this being renamed? And why is this change in a commit labelled as a package fix?